config.json

```diff
@@ -1353,17 +1353,17 @@
       {
         "groupId": "com.google.dagger",
         "artifactId": "dagger",
-        "version": "2.41",
-        "nugetVersion": "2.41.0",
+        "version": "2.42",
+        "nugetVersion": "2.42.0",
         "nugetId": "Xamarin.Google.Dagger",
         "dependencyOnly": false,
         "templateSet": "dagger"
       },
       {
         "groupId": "com.google.errorprone",
         "artifactId": "error_prone_annotations",
-        "version": "2.10.0",
-        "nugetVersion": "2.10.0",
+        "version": "2.7.1",
```

Comment:
Many of these dependencies are going backwards.
- Xamarin.Google.ErrorProne.Annotations
- Xamarin.Protobuf.JavaLite
- Xamarin.Grpc.Android